form.html

```diff
@@ -0,0 +1,159 @@
+                  </div>
+                </li>
+                <li class="form__radio-item">
+                  <div class="form__negative-answer">
+                    <input class="form__input-radio" id="answer-negative" name="answer" type="radio" value="rather-negative"/>
+                    <label class="form__label-radio" for="answer-negative">Rather negative</label>
+                  </div>
+                </li>
+                <li class="form__radio-item">
+                  <div class="form__difficult-answer">
+                    <input class="form__input-radio" id="answer-difficult" name="answer" type="radio" value="difficult-answer"/>
+                    <label class="form__label-radio" for="answer-difficult">Difficult to answer</label>
+                  </div>
+                </li>
+              </ul>
+            </fieldset>
+          </section>
+
+          <section class="form__section">
+            <h2 class="form__header">Sights visited:</h2>
+            <ul class="form__checkbox-list">
+              <li class="form__checkbox-item">
+                <div class="form__visited-places">
+                  <input class="form__input-checkbox" id="bridge" name="bridge" type="checkbox" value="devils-bridge">
+                  <label class="form__label-checkbox" for="bridge">The Devil's Bridge</label>
+                </div>
+              </li>
+              <li class="form__checkbox-item">
+                <div class="form__visited-places">
+                  <input class="form__input-checkbox" id="park" name="park" type="checkbox" value="slide-park">
+                  <label class="form__label-checkbox" for="park">Slide-park</label>
+                </div>
+              </li>
+              <li class="form__checkbox-item">
+                <div class="form__visited-places">
+                  <input class="form__input-checkbox" id="mountain" name="mountain" type="checkbox" value="mountain-bell">
+                  <label class="form__label-checkbox" for="mountain">Mountain-bell</label>
+                </div>
+              </li>
+              <li class="form__checkbox-item">
+                <div class="form__visited-places">
+                  <input class="form__input-checkbox" id="cliffs" name="cliffs" type="checkbox" value="red-cliffs">
+                  <label class="form__label-checkbox" for="cliffs">Red cliffs</label>
+                </div>
+              </li>
+            </ul>
+          </section>
+
+          <section class="form__section">
+            <div class="form__feedback">
+              <label class="form__label-textarea" for="feedback">Describe your emotions:</label>
+              <textarea class="form__input-textarea" id="feedback" name="feedback"></textarea>
+              <p class="form__paragraph--shaddow">* - required fields</p>
+            </div>
+          </section>
+
+          <section class="form__btn">
+            <button class="form__submit">POST REVIEW</button>
```

Comment:
if it's a submit button for a form I think you need to set a type="submit"
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button